scripts/build_mod_ic.py

```diff
@@ -112,6 +112,17 @@
         "pin_shape": (0.95, .30),
     },
 
+    # LQFP-32 from JEDEC MS-026BBA
+    "LQFP-32": {
+        "rows": 4,
+        "pins": 32,
+        "pin_pitch": 0.8,
+        "row_pitch": 8.4,
+        "pad_shape": (1.5, 0.5),
```

Comment:
The IPC-7351B recommendation for QFP80P900X900X160-32N is 1.35x0.50 (vs ST's 1.2x0.5) but 1.5 is definitely out :P (feel free to add the IPC-7351B part name too).